src/FactoryPattern/FactorySelection.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace FactoryPattern
+{
+    class FactorySelection
+    {
+        public static CarModelFactory CreateCarFactory(string FactoryType)
+        {
+            if (FactoryType.ToLower() == "electric")
+            {
+                System.Console.WriteLine("entrel meqenaneric meke |Tesla,Nissan,Jaguar|");
```

Comment:
avoid printing on console things from classes. They can be used in web app and it writeline will break the class. Print only in main method.